examples/keyboard_copy_paste.cpp

```diff
@@ -0,0 +1,32 @@
+#include <macro/macro.h>
+
+#include <iostream>
+
+int main() {
+    Macro::Misc::Sleep(2000);
+    
+    std::cout << "Copy Me Once and Paste Me Twice!";
+
+    Macro::Misc::Sleep(400);
+    Macro::Keyboard::Down(Macro::Keyboard::Key::LCTRL);
+    Macro::Keyboard::Tap(Macro::Keyboard::Key::A);
+    Macro::Keyboard::Up(Macro::Keyboard::Key::LCTRL);
+
+    Macro::Keyboard::Down(Macro::Keyboard::Key::LCTRL);
+    Macro::Keyboard::Tap(Macro::Keyboard::Key::INSERT, 40);
+    Macro::Keyboard::Up(Macro::Keyboard::Key::LCTRL);
```

Comment:
This example is a bit different from the other ones since it copies a chunk of text.

I think it should output `Copy Me Once and Paste Me Twice!`, wait for a tiny bit, then select all and copy the text without the 2 second delay at the start. After it copies the text, it should wait for around 2s to allow the user to switch tabs and then paste it twice.